style-guide.md

```diff
@@ -0,0 +1,189 @@
+
+A job title should be brief because it is often used as the heading of a
+document or page. It is the first thing a candidate will notice and attracts
+them to your job description.
+
+A job title is a brief description of the overall type of job you are looking
+for. It needs to be brief because it is often used as the title of the document
+and it is good to be descriptive because often this will be the first bit of
+information a candidate sees in a search result. The more attention grabbing the
+better.
+
+#### Examples
+
+"Web Developer" is fine but... "Front-End Developer" or "Full-Stack Developer" is better.
+
+
+Do mention the main language or framework if possible. 
+
+"Ruby Web Developer" or "Rails Developer"
+
+Don't use internal job titles such as "JAVA DEVELOPER-3", no one outside your
+company will know what that means, and it will likely not match any candidates
+search criteria or standout in the search results.
+
+Don't use "Rockstar" or "Ninja", it is off putting to most professionals now to
+be referred to as such at least in the cincytech slack community.
+
+### Company Description
+
+A company description is a paragraph describing the domain the company operates
+in. It may also include the culture/values of the company. It does not
+necessarily need to mention the name of the company but that would be ideal.
+
+A company description is necessary for a couple of reasons.
+
+1. It implies the nature of the work. 
+
+If you're a design agency, your work is likely project based and for external clients. Some people prefer that type of work, some do not. 
+
+If you're a startup, the culture is likely vastly different than that of a financial institution. Some people may prefer the excitement and risk of working for a startup, some may prefer the stability and culture of a bank. 
+
+
+2. It helps save you and the candidates time. 
+
+
+#### Examples
+
+"We are a startup working to make the internet of things more secure."
+
+Do mention location...
+
+"We are a startup in Cincinnati Ohio working to make the internet of things more secure."
+
+Do provide a link for more information if possible.
+
+"We are a startup in Cincinnati Ohio working to make the internet of things more secure. [find out more](http://google.com)"
+
+Don't just copy/paste your mission statement. Those are rarely understood outside the company. 
+
+Don't just leave out the company description.
+
+### Compensation
+
+The compensation section should include whether the part time/full time/contract/volunteer and list a range of salary or hourly rate. Benefits offered is also good.
+
+Even if the rate is negotiable, it's better to be upfront about what you are offering for the position. Lots of time and effort can be wasted interviewing only to find out that your compensation expectations and the candidates are impossible to align. 
+
+
+#### Examples
+
+"This is a full time position with a salary range of 50K-75K, we also offer health dental and vision"
```

Comment:
Do you think it would also be good to know what percentage of health care is covered by the employer or a general idea of what the monthly premium is? I know I have had some jobs where I have had to pay $600 a month and some where its $100. That really impacts take home and overall salary negotiations IME